backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/database/entity/VersionEntity.kt

```diff
@@ -21,9 +21,9 @@ data class VersionEntity(
         @EmbeddedId @IndexedEmbedded val versionId: VersionId,
         @Field @Column(columnDefinition = "TEXT") val content: String,
         @Field @Column(columnDefinition = "TEXT") val title: String,
-        @Field @Column(columnDefinition = "TEXT") val description: String,
+        @Field @Column(columnDefinition = "TEXT") val description: String?,
         @Field @Column(columnDefinition = "TEXT") val language: ApiLanguage,
-        @Field @Column(columnDefinition = "TEXT") val servers: String,
+        @Field @Column(columnDefinition = "TEXT") val server: String?,
```

Comment:
One last idea: How about calling this "endpoint" instead?

Reply:
endpointUrl, to make clear what the contents represent?